loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowParameters.java

```diff
@@ -125,7 +126,8 @@ protected static void load(LoadFlowParameters parameters, PlatformConfig platfor
                 .ifPresent(config -> {
                     parameters.setVoltageInitMode(config.getEnumProperty("voltageInitMode", VoltageInitMode.class, DEFAULT_VOLTAGE_INIT_MODE));
                     parameters.setTransformerVoltageControlOn(config.getBooleanProperty("transformerVoltageControlOn", DEFAULT_TRANSFORMER_VOLTAGE_CONTROL_ON));
-                    parameters.setNoGeneratorReactiveLimits(config.getBooleanProperty("noGeneratorReactiveLimits", DEFAULT_NO_GENERATOR_REACTIVE_LIMITS));
+                    parameters.setNoGeneratorReactiveLimits(!config.getBooleanProperty("noGeneratorReactiveLimits", !DEFAULT_USE_REACTIVE_LIMITS)); // overwritten by reactiveLimits
```

Comment:
Not sure why you set `!config...`?

Reply:
About this, can you explain? There is two negations so I think it is a mistake

Reply:
A merge mistake. Fixed.